### ISSUE TITLE
Script for training TSM on something-something-v2

### ISSUE THREAD

**Issue by pplntech** [mit-han-lab/temporal-shift-module#21]
Hi Ji, thank you for publishing the work.

I want to double-check training parameters for training TSM on something-v2 which should achieve at least 58.8 which is the performance I got when testing with your weight. (tested with single crop and single clip)

According to your paper, the training parameters for the something-something-v2 dataset are: 50 training epochs, initial learning rate 0.01 (decays by 0.1 at epoch 20&40), weight decay 1e-4, batch size 64, and dropout 0.5. And the model is fine-tuned from ImageNet pre-trained weights.

However, the script in the git repository indicates that initial learning rate is 0.001, weight decay is 5e-4, and the model is tuned-from Kinetics pre-trained weight.

Due to this disparity, I am confusing which parameters should I use to reproduce the number. Could you provide accurate parameters for training TSM on something-v2?

Thank you.

**Comment by tonylins** [comment]
Hi, sorry for the confusion. You should use the setting in the paper. I may forget to update the script in this repo. The old setting tuning from Kinetics will produce inferior result.
